Shared/DataTransferObjects/AttendanceDto.cs

```diff
@@ -0,0 +1,9 @@
+namespace Shared.DataTransferObjects;
+
+public record AttendanceDto(
+    Guid Id, 
+    Guid EmployeeId, 
+    DateTime CheckInTime, 
+    DateTime? CheckOutTime, 
+    string Status, 
+    string? Notes);
```

Comment:
To improve naming consistency and intent, consider renaming `Notes` to `Note` unless it is explicitly a collection of multiple values. Using singular form aligns better with DTO conventions when handling single text fields.

**Suggested change**
```suggestion
    string? Note);
```